mlir/test/fusion/e2e/resnet50/lit.local.cfg

```diff
@@ -1,2 +1,3 @@
-if not config.enable_rock_driver_e2e_test or config.no_AMD_GPU:
-  config.unsupported = True
+#if not config.enable_rock_driver_e2e_test or config.no_AMD_GPU:
+#  config.unsupported = True
+config.unsupported = False
```

Comment:
... why this change?

Reply:
Yes, I somehow committed the wrong file. That was me shortcut our lit file to make it test a larger subset when doing quantization work...

Corrected now.